swerve100/src/main/java/frc/robot/subsystems/SwerveDriveSubsystem.java

```diff
@@ -438,10 +438,13 @@ public void drive(double xSpeed, double ySpeed, double rot, boolean fieldRelativ
         if (Math.abs(rot) < .01)
             rot = 0;
         double gyroRate = m_gyro.getRate() * 0.25;
-        Rotation2d rotation2 = getPose().getRotation().minus(new Rotation2d(gyroRate));
+        // Rotation2d rotation2 = getPose().getRotation().minus(new Rotation2d(gyroRate));
```

Comment:
is it essential to remove the rotation/translation coupling correction?  maybe instead of removing it you could tune the coefficient (0.25)?  @Coolearthsky  might have an opinion about that?